src/developer/web-api/tracker.md

```diff
@@ -550,6 +556,21 @@ Examples for the **FLAT** and the **NESTED** versions of the payload are listed
 
 #### ***NESTED*** payload
 
+Nested payloads are the most commonly used structure, where tracker objects are embedded within
+their parent objects, such as an enrollment within a tracked entity. The advantage of this structure
+is that the client does not need to provide UIDs for these references, as this is handled
+automatically.
+
+> **NOTE**
+>
+> Although nested payloads can be easier for clients to manage, the payload will always be flattened
+> before the import. For large imports, using a flat structured payload offers more control and
+> reduces overhead during the import process.
+> 
+> That being said, you cannot use nested payloads to create relationships. This is because it is not
```

Comment:
`you cannot use nested payloads to create relationships`.
We should be more clear here, we can create relationships in nested payloads, like a payload with 2 TEs and one relationship nested in a TE will work.
What is not possible is to have new entities embedded in the relationship, hence relationships are always leaves of the payload